content/pages/about.html

```diff
@@ -25,7 +25,13 @@ <h2>About NeuroData</h2>
-            <p>To flourish together by extending and fusing statistical machine learning and big data science to address the most important brain science and mental health questions of our time.</p>
+            <p>Our central motivation is to understand intelligence, both animal and machine, and in particular how learning and memory arises in such systems.  To do so, we design, build, study, and apply statistical machine learning and big data science techniques.  The tools we develop and utilize follow some basic design principles, including philosophical  (validity, uncertainty), statistical (consistency, efficiency, robustness), and computational (scalability, tunability). While we have no idea how intelligence works, we do have some current working hypotheses, subject to revision given new data or theories.  </p>
+
+            <p>For animal intelligence, our current conjecture is that to a first approximation, intelligence arises largely due to communication between disparate entities with specialized properties across multiple scales. In other words, we believe that the key to understanding various animal intelligences is determining which connections between which entities with which properties at which scales are the mechanism underlying the intelligences.  To answer these questions, we collaborate extensively with some of the best experimental neuroscientists in the world at different scales, each of whom design experiments and collect data amenable to answering these questions.  We then design, build, study, and apply statistical models and estimators designed to reveal the latent structures in these big connectome datasets. Because the raw data are never in a form amenable to directly study, we also build big data systems to manage, visualize, and wrangle the data. </p>
+
+            <p>For machine intelligence we focus on studying the foundations of learning and memory.  We believe that both animal and machine intelligence operates under certain shared principles, common to any intelligent system.  Only by virtue of understanding these principles do we have hope that we can find the biological mechanisms implementing these principles.  While we lack clarity on what are the key principles, we  believe that "deep learning" is <em>not</em> a key principle for intelligence, rather, deep learning is more likely to be a key principle for optimization.   However, as mentioned above, we are open to being wrong about this, and will continue to  collaborate with deep learning practitioners, while we study principles that we believe to be foundational.  One  principle that we do believe is a <em>key</em> principle is estimating geodesic distances learning, that is, the ability to leverage data from disparate tasks (past, present, and future) to improve other tasks.  We also believe that satisfactorily estimating latent geodesic distances will be a crucial component any of efficient learning machines. </p>
```

Comment:
I'd like to discuss this more, possibly as a group, because I don't know that I agree with everything here. Or at least I don't feel like it's true in practice when we are working on "machine intelligence." Also could probably be more succinct

Reply:
make a PR.  PR's get integrated unless they are the wrong direction.  if you think this is worse than the existing, that's one thing.  if you think it is an improvement, and you want to further refine, that's another.  please clarify.  

also, we have about 15 papers in PAMI, where the MI stands for "machine intelligence".  i'd say almost all of our work is machine intelligence.